### PR TITLE
Update Package-Restore.md to mention and link to one command to restore and build with msbuild

### PR DESCRIPTION
**Pull request by rrelyea**

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[13c5a0a](https://github.com/NuGet/docs.microsoft.com-nuget/commits/13c5a0a06d69933ef5d1296eed9daef00dfb692d)_: 

:clock10: **Pending**: waiting for processors ([6](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 6 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[13c5a0a](https://github.com/NuGet/docs.microsoft.com-nuget/commits/13c5a0a06d69933ef5d1296eed9daef00dfb692d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Package-Restore.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/rrelyea-oneCommandToRestoreAndBuild/docs/consume-packages/Package-Restore.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/package-restore?branch=pr-en-us-1969) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C24%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202003242228031571-1969%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=QxEeNVUUwQd623dPJytTU4FfWSUak7RDxIglWhrBNFg%3D&st=2020-03-24T22%3A23%3A40Z&se=2020-04-24T22%3A28%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/consume-packages/Package-Restore.md [hunk 1]
```diff
@@ -162,6 +162,14 @@ To restore packages listed in the project file with PackageReference, use the th
    ```
 
    Make sure that the MSBuild output indicates that the build completed successfully.
+   
+> [!Note]
+> msbuild has a `-restore` switch which will run `Restore` and then build. See [Restoring and building with one MSBuild command](/nuget/reference/msbuild-targets#restoring-and-building-with-one-msbuild-command).
```
Comment (zivkan) [inline docs/consume-packages/Package-Restore.md]: It's more than just running restore and the build, it reloads certain things, meaning `-restore` is different to `-t:restore,build` or `-t:restore -t:build`.

Comment (karann-msft) [reply docs/consume-packages/Package-Restore.md]: @zivkan - can you suggest updates to address your feedback? Use the "suggestions" feature :) 
image

Comment (zivkan) [reply docs/consume-packages/Package-Restore.md:167]:
```suggestion
> msbuild has a `-restore` switch which will run `Restore`, reload the project, and then build. See [Restoring and building with one MSBuild command](/nuget/reference/msbuild-targets#restoring-and-building-with-one-msbuild-command).
```

**Comment by anangaur** [comment]
Docs Build status updates of commit _[aef58a3](https://github.com/NuGet/docs.microsoft.com-nuget/commits/aef58a3069439407aa54ed7f4bcc6900e80c0cff)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Package-Restore.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/rrelyea-oneCommandToRestoreAndBuild/docs/consume-packages/Package-Restore.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/package-restore?branch=pr-en-us-1969) | [Details](#user-content-9251b79c-2da2e837)

<a id="9251b79c-2da2e837"></a>
### [docs/consume-packages/Package-Restore.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/rrelyea-oneCommandToRestoreAndBuild/docs/consume-packages/Package-Restore.md)
 - **Line 5, Column 12**: **[Warning-[ms-author-invalid](https://review.docs.microsoft.com/en-us/help/contribute/validation-ref/ms-author-invalid?branch=main)]** `````Invalid value for 'ms.author', 'karann' is not a valid Microsoft alias.`````
 - **Line 176, Column 1**: **[Suggestion-preserve-view-not-set]** `````Did you mean to link to a specific version? If so, add &preserve-view=true to the URL: /azure/devops/pipelines/tasks/build/dotnet-core-cli?view=azure-devops NOTE: This Suggestion will become a Warning on 1/29/21.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202010192256069666-1969%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Uwnzd%2B89cgMfgRiv3hRRA7scrntSVqV0uihURGrJIJ0%3D&st=2020-10-19T22%3A51%3A44Z&se=2020-11-19T22%3A56%3A44Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://teams.microsoft.com/l/message/19:7ecffca1166a4a3986fed528cf0870ee@thread.skype/1590030602688?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&parentMessageId=1590030602688&teamName=Docs%20Support&channelName=General&createdTime=1590030602688) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by zivkan — APPROVED** [review]

**Pull request merged**